### PR TITLE
Add California dates

### PR DESCRIPTION
**Pull request by tannewt**

### CODE REVIEW

File: states/california/elections.toml [hunk 1]
```diff
@@ -7,3 +7,24 @@ original_date = 2020-03-03
 date = 2020-11-03
 name = "General"
 original_date = 2020-11-03
+
+[20201103.registration]
+online_by = 2020-10-19
+in_person_by = 2020-11-03
+postmarked_by = 2020-10-19
+
```
Comment (askpatrickw) [inline states/california/elections.toml:16]:
```suggestion
sources = ["https://elections.cdn.sos.ca.gov//statewide-elections/2020-primary/section-8-general-calendar.pdf"]
```

**Comment by tannewt** [comment]
11/20 received by deadline confirmed for mail-in-ballots.

Screen Shot 2020-07-10 at 1 08 42 PM

File: states/california/elections.toml [hunk 2]
```diff
@@ -7,3 +7,24 @@ original_date = 2020-03-03
 date = 2020-11-03
 name = "General"
 original_date = 2020-11-03
+
+[20201103.registration]
+online_by = 2020-10-19
+in_person_by = 2020-11-03
+postmarked_by = 2020-10-19
+sources = ["https://elections.cdn.sos.ca.gov//statewide-elections/2020-primary/section-8-general-calendar.pdf"]
+[20201103.absentee]
```
Comment (askpatrickw) [inline states/california/elections.toml:17]:
```suggestion

[20201103.absentee]
```

Comment (askpatrickw) [reply states/california/elections.toml:17]: super nitpicky... whitespace.

**Review by askpatrickw — CHANGES_REQUESTED** [review]
Looks good aside from the nice-to-have link to the source (which is already in there) and the nitpicky whitespace which I messed up with my suggested change. ;-)

**Comment by tannewt** [comment]
Please take one more look!

**Review by askpatrickw — APPROVED** [review]
Looks good!

**Pull request merged**